web-app/src/api/translator.js

```diff
@@ -0,0 +1,25 @@
+import api from ".";
+
+export const formatReport  = async({messages, full}) => {
+    const { timestamp, author, content } = messages[0].value;
+    const { node, common_issue, title, body}  = content
+    const user = await api.account.get(author);
+    return {
+        id: messages[0].key,
+        timestamp,
+        user,
+        status: 'open',
+        node,
+        commonIssueId: common_issue.id,
+        title,
+        body,
+        comments: messages.splice(0,1).map(msg => msg.key)
```

Comment:
This is exploratory or the intended final behavior? In that case it would be nice to avoid mutating 'messages' with splice, isn't it?